guide/sections/part2/wis2node.adoc

```diff
@@ -83,25 +92,56 @@ Third, the Open Geospatial Consortium (OGC) have developed a suite of APIs (call
 
 Finally, we’re increasingly concerned with providing access to very large Datasets. The OGC has published a series of informative blogs on the subject of cloud-native geospatial data sharing. These are listed among in section 11.4.2 Informative References.
 
-TODO: to be completed
 
 ====== Publication and topic selection
 
 When publishing a dataset, a data publisher selects a given topic according to the WIS Topic Hierarchy.  Given the multidisciplinary nature of some data, a data publisher must select a single topic for publication purposes, and always uses WCMP2 discovery metadata to provide a fulsome description of their dataset and its relevance to additional disciplines.
 
+Metadata is the method by which datasets are ultimately made available in the WIS2 system.  The goal is for data providers who have PR authorization to have a lightweight method to provide their datasets to WIS.  With this goal in mind, there are several acceptable methods to publish metadata:  
+
+    Option 1: deploy a WIS2 node  
+
+    Option 2: a MQTT broker and HTTP server  
+
+    Option 3: a bilateral agreemnt for another organization to publish metadata publication on behalf of the data provider (potential organizations providing this service are GISCs and NMHS or potentional through a WIS2 portal in the future). 
+
+
+For infrequently updated datasets the following process should be followed:  
+
+    Publish initial metadata.  
+    
+    Publish update metadata.  
+    
+    Data update notification: normal notification message with property.cache=false  
+
+Use of the "experimental" topic  
+
+    The "experimental" topic is necessary for the WIS2 pre-operational phase and future pre-operational data exchange in test mode.
```

Comment:
experimental topic may also be necessary when the sub-topics have not been approved yet